CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 0.16.0
+
+### New Features
+
+- Expose new `render`, `documentRender` and `documentRenderAsString` methods
```

Comment:
Aren't they called `serverRender` and `serverRenderToString` 🤔 ?

Reply:
Woke up, but still sleeping :) Thanks @Meemaw